sweph/src/Makefile

```diff
@@ -27,7 +27,7 @@ SWEOBJ = swedate.o swehouse.o swejpl.o swemmoon.o swemplan.o sweph.o\
 	 swephlib.o swecl.o swehel.o
 
 swetest: swetest.o libswe.a
-	$(CC) $(OP) -o swetest swetest.o -L. -lswe -lm -ldl
+	$(CC) $(OP) -o swetest swetest.o -L. libswe.a -lm -ldl
```

Comment:
Will this be something we have to remember to manually change each time we "drop in" updated SE source code?

Reply:
I have reverted this change